include/pybind11/stl_bind.h

```diff
@@ -615,6 +635,12 @@ class_<Map, holder_type> bind_map(handle scope, const std::string &name, Args&&.
     }
 
     Class_ cl(scope, name.c_str(), pybind11::module_local(local), std::forward<Args>(args)...);
+    class_<KeysView> keys_view(
+        scope, ("KeysView[" + name + "]").c_str(), pybind11::module_local(local));
+    class_<ValuesView> values_view(
+        scope, ("ValuesView[" + name + "]").c_str(), pybind11::module_local(local));
+    class_<ItemsView> items_view(
+        scope, ("ItemsView[" + name + "]").c_str(), pybind11::module_local(local));
```

Comment:
This naming seems to produce "erroneous" names for stub generation and subsequent parsing by Pylance or mypy, etc... Also sphinx autodoc has issues parsing it:
```
reading sources... [ 40%] references/python
WARNING: invalid signature for autoclass ('depthai::BoardConfig.ItemsView[GPIOMap]')
```

Thoughts on renaming to `*View_{name}`, with an underscore? Seems to work well afterwards.

Example
Before:
```
class GPIOMap:
    def __init__(self) -> None: ...
    def items(self) -> BoardConfig.ItemsView[GPIOMap]: ...
    def keys(self) -> BoardConfig.KeysView[GPIOMap]: ...
    def values(self) -> BoardConfig.ValuesView[GPIOMap]: ...
    def __bool__(self) -> bool: ...
    @overload
    def __contains__(self, arg0: int) -> bool: ...
    @overload
    def __contains__(self, arg0: object) -> bool: ...
    def __delitem__(self, arg0: int) -> None: ...
    def __getitem__(self, arg0: int) -> BoardConfig.GPIO: ...
    def __iter__(self) -> Iterator: ...
    def __len__(self) -> int: ...
    def __setitem__(self, arg0: int, arg1: BoardConfig.GPIO) -> None: ...

class ItemsView[GPIOMap]:
    def __init__(self, *args, **kwargs) -> None: ...
    def __iter__(self) -> Iterator: ...
    def __len__(self) -> int: ...
...
```

After:
```
class GPIOMap:
    def __init__(self) -> None: ...
    def items(self) -> BoardConfig.ItemsView_GPIOMap: ...
    def keys(self) -> BoardConfig.KeysView_GPIOMap: ...
    def values(self) -> BoardConfig.ValuesView_GPIOMap: ...
    def __bool__(self) -> bool: ...
    @overload
    def __contains__(self, arg0: int) -> bool: ...
    @overload
    def __contains__(self, arg0: object) -> bool: ...
    def __delitem__(self, arg0: int) -> None: ...
    def __getitem__(self, arg0: int) -> BoardConfig.GPIO: ...
    def __iter__(self) -> Iterator: ...
    def __len__(self) -> int: ...
    def __setitem__(self, arg0: int, arg1: BoardConfig.GPIO) -> None: ...

class ItemsView_GPIOMap:
    def __init__(self, *args, **kwargs) -> None: ...
    def __iter__(self) -> Iterator: ...
    def __len__(self) -> int: ...
...
```

Reply:
@themarpe sounds good. Please add a PR.

Reply:
@themarpe Actually, hold on. Why is it having any issues? It should be generating a typing.KeysView . Is that why it's struggling?

Reply:
@Skylion007 Thanks - will do so soon.

I think its mainly from the `*View[name]` naming. I think the square brackets gets at it. Works okay for mypy & Sphinx when renamed to something different